apps/prairielearn/elements/pl-matching/pl-matching.mustache

```diff
@@ -4,7 +4,13 @@
 {{#statements}}
     <div class="pl-matching-statement">
         <div class="pl-matching-statement-select">
-            <select name="{{name}}" class="form-select" {{^editable}}disabled{{/editable}} aria-labelledby="statement-label-{{statement_id}}">
+            <select
+                name="{{name}}"
+                class="form-select"
+                {{^editable}}disabled{{/editable}}
+                {{#aria-label}}aria-label="{{aria-label}}"{{/aria-label}}
+                {{^aria-label}}aria-labelledby="statement-label-{{statement_id}}"{{/aria-label}}
+                >
```

Comment:
```suggestion
            >
```

Reply:
I'm really hoping we can get a proper formatter for the files. I looked into this last night, and the best approach I think is to modify a django-html tree-sitter grammar to get a proper parse. https://github.com/mja129/tree-sitter-htmldjango/blob/master/grammar.js